modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase

```diff
@@ -0,0 +1,52 @@
+@component-name = "OSB Site Initializer Partner Portal"
+definition {
+
+	var baseURL = PropsUtil.get("portal.url");
+
+	property custom.properties = "feature.flag.LPS-135430=true";
+	property osgi.modules.includes = "osb-site-initializer-partner-portal";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "OSB Site Initializer Partner Portal";
+
+	setUp {
+		task ("Set up instance and sign in") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Create a new Partner Portal Site") {
+			PRMUtils.addPRMSite();
+		}
+
+		PRMUtils.gotoPRM();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Partner Portal");
+		}
+	}
+
+	@description = "This is a test for LPS-171642. User can access MDF Request Listing through Quick Links"
+	@priority = "4"
+	test CanAccessMDFRequestListingViaQuickLinks {
+		task ("When I click on the MDF") {
+			Click(locator1 = "PRMHomePage#CLICK_MDF");
+		}
```

Comment:
Hi @juliosilveira77, I think that this task could be something like: "When I click on 'MDF' button, on Quick Links section". What do you think? On this way the task is more clear for who is reading it.